charts/prom-rds-exporter/values.yaml

```diff
@@ -0,0 +1,233 @@
+  #     replacement: mydbname
+  #     targetLabel: dbname
+
+prometheusRule:
+  # Specifies whether a PrometheusRule should be created
+  enabled: false
+  # Set the namespace the PrometheusRule should be deployed
+  # namespace: monitoring
+  # Set labels for the PrometheusRule, use this to define your scrape label for Prometheus Operator
+  # labels:
+  # Example - note the Kubernetes convention of camelCase instead of Prometheus'
+  # rules:
+  #    - alert: RDSEnhancedHighCPU
+  #      annotations:
+  #        message: RDS instance {{`{{`}} $labels.exported_instance {{`}}`}} in region {{`{{`}} $labels.region{{`}}`}} is experiencing high CPU over 70%.
+  #      expr: rdsosmetrics_cpuUtilization_total{job="prom-rds-exporter"} >= 70
+  #      for: 1m
+  #      labels:
+  #        severity: critical
+  #    - alert: RDSEnhancedHighMemory
+  #      annotations:
+  #        message: RDS instance {{`{{`}} $labels.exported_instance {{`}}`}} in region {{`{{`}} $labels.region{{`}}`}} is using over 70% memory.
+  #      expr: (node_memory_Active_bytes{job="prom-rds-exporter"})/(node_memory_MemTotal_bytes{job="prom-rds-exporter"})*100 >= 70
+  #      for: 1m
+  #      labels:
+  #        severity: critical
+  #    - alert: RDSEnhancedHighRootPartition
+  #      annotations:
+  #        message: RDS instance {{`{{`}} $labels.exported_instance {{`}}`}} in region {{`{{`}} $labels.region{{`}}`}} is using more than 70% of root partition /.
+  #      expr: 100-((node_filesystem_avail_bytes{job="prom-rds-exporter",mountpoint="/"})/(node_filesystem_size_bytes{job="prom-rds-exporter",mountpoint="/"})*100) >= 70
+  #      for: 1m
+  #      labels:
+  #        severity: critical
+  #    - alert: RDSEnhancedHighDataPartition
+  #      annotations:
+  #        message: RDS instance {{`{{`}} $labels.exported_instance {{`}}`}} in region {{`{{`}} $labels.region{{`}}`}} is using more than 70% of data partition /rdsdbdata.
+  #      expr: 100-((node_filesystem_avail_bytes{job="prom-rds-exporter",mountpoint="/rdsdbdata"})/(node_filesystem_size_bytes{job="prom-rds-exporter",mountpoint="/rdsdbdata"})*100) >= 70
+  #      for: 1m
+  #      labels:
+  #        severity: critical
+    #    - alert: RDSEnhancedHighLoadAverage
+  #      annotations:
+  #        message: RDS instance {{`{{`}} $labels.exported_instance {{`}}`}} in region {{`{{`}} $labels.region{{`}}`}} is having a high Load Average.
+  #      expr: rdsosmetrics_loadAverageMinute_one{job="prom-rds-exporter"} >= rdsosmetrics_General_numVCPUs{job="prom-rds-exporter"}
+  #      for: 1m
+  #      labels:
+  #        severity: critical
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+securityContext:
```

Comment:
securityContext is duplicated

Reply:
Hi @egegunes 
Thanks for comment. I've updated the chart and also check `helm lint` from my locally. Please verify again. Thanks.